2-WebApp-graph-user/2-1-Call-MSGraph/Properties/launchSettings.json

```diff
@@ -12,15 +12,17 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "KEY_VAULT_URI": "https://keyvault0417.vault.azure.net/"
```

Comment:
This is to be able to debug locally, given that the code in GetSecretFromKeyVault looks for an environment variable (we could do differently)